modules/administration_manual/pages/installation/manual_installation_ubuntu_16_04.adoc

```diff
@@ -0,0 +1,163 @@
+= Manual Installation on Linux - Ubuntu 16.04
+:nginx-app-tracing-url: https://www.nginx.com/blog/application-tracing-nginx-plus/
+
+== Introduction
+This guide describes how to install the required dependencies, the database and ownCloud itself in a
+single server setup.
+
+
+[[install-the-required-packages]]
+== Prepare your Server
+
+*Necessary steps to prepare the installation of ownCloud*
+
+Before we can start to install ownCloud, we need to make sure that everything needed
+is available.
+
+Steps to take:
+
+* Install MariaDB Database
+* Install Redis Server
+* Install Apache and the necessary dependencies
+* Configure Apache
+* Download and install ownCloud 
+
+
+=== Install MariaDB Database
+
+Perform the following steps to install MariaDB and setting up a Database for ownCloud.
+
+....
+apt update
+apt install -y mariadb-server-10.0
+....
+After this has finish, create the database and a database user for ownCloud to use:
+
+....
+mysql -u root
+
+MariaDB [(none)]> create database owncloud;
```

Comment:
this can be done with one line.

```
mysql -u root -e "CREATE DATABASE IF NOT EXISTS owncloud; \
GRANT ALL PRIVILEGES ON owncloud.* TO admin@localhost IDENTIFIED BY 'admin'";
```